File: client/src/main/java/com/mthwate/conk/listener/AbstractClientListener.java
package com.mthwate.conk.listener;

import com.jme3.network.Client;
import com.jme3.network.Message;
import com.mthwate.conk.message.AbstractMessageListener;

/**
 * @author mthwate
 */
public abstract class AbstractClientListener<T extends Message> extends AbstractMessageListener<Client, T> {}

File: client/src/main/kotlin/com/mthwate/conk/listener/AbstractClientListener.kt
package com.mthwate.conk.listener

import com.jme3.network.Client
import com.jme3.network.Message
import com.mthwate.conk.message.AbstractMessageListener

/**
 * @author mthwate
 */
abstract class AbstractClientListener<T : Message> : AbstractMessageListener<Client, T>()
